food-truck-frontend/src/components/util/multi_field.tsx

```diff
@@ -15,8 +19,9 @@ type MultiState = {
 class MultiField extends Component<MultiProps, MultiState> {
   constructor(props: MultiProps) {
     super(props);
+    console.log(props);
```

Comment:
We'll probably want to remove this debug print. That's on me for forgetting, though.